services/apps/alcs/src/alcs/inquiry/inquiry-document/inquiry-document.entity.ts

```diff
@@ -8,11 +8,13 @@ import {
   ManyToOne,
   OneToOne,
   PrimaryGeneratedColumn,
+  Unique,
 } from 'typeorm';
 import { DocumentCode } from '../../../document/document-code.entity';
 import { Document } from '../../../document/document.entity';
 import { Inquiry } from '../inquiry.entity';
 
+@Unique('UNIQUE_OATS_ISSUE_IDS', ['oatsDocumentId', 'oatsIssueId'])
```

Comment:
Will this break regular document uploads from ALCS since we are not populating these values?